tests/countries/test_india.py

```
@@ -1690,17 +2046,22 @@ def test_2018_optional(self):
             ("2018-01-14", "Magh Bihu; Makar Sankranti; Pongal; Uttarayan"),
             ("2018-01-15", "Thiruvalluvar Day / Mattu Pongal"),
-            ("2018-01-16", "Uzhavar Thirunal"),
+            ("2018-01-16", "Feast of St. Joseph Vaz; Uzhavar Thirunal"),
+            ("2018-01-20", "Gend Singh's Shaheedi Diwas"),
+            ("2018-01-21", "Hemu Kalani's Shaheedi Diwas"),
             (
                 "2018-01-22",
-                "Basant Panchami / Shri Panchami; "
-                "Satguru Ram Singh's Jayanti; "
+                "Basant Panchami / Shri Panchami; Satguru Ram Singh's Jayanti; "
                 "Sir Chottu Ram's Jayanti",
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)